autoload/codefmt/jsbeautify.vim

```diff
@@ -30,7 +30,7 @@ function! codefmt#jsbeautify#GetFormatter() abort
   endfunction
 
   function l:formatter.AppliesToBuffer() abort
-    return &filetype is# 'css' || &filetype is# 'html' || &filetype is# 'json' ||
+    return &filetype is# 'css' || &filetype =~ 'html.*' || &filetype is# 'json' ||
         \ &filetype is# 'javascript'
```

Comment:
This loose matching includes some patterns you might not have intended. Consider changing it to something like
```vim
&filetype is# 'html' || &filetype =~# '\mhtml\.'
```

Without the `#` the behavior subtly depends on the `'ignorecase'` setting, and the loose `.*` will match on characters that aren't a literal dot, such as some weird `htmlos` filetype vim has that doesn't seem to be anything like actual HTML syntax.